operating/manage-vulnerabilities/scan-rhcos-node-host.adoc

```diff
@@ -17,7 +17,7 @@ First, {product-title-short} analyzes and detects RHCOS components. Then it matc
 [NOTE]
 ====
 * If you installed {product-title-short} by using the `roxctl` CLI, you must manually enable the RHCOS node scanning features. When you use Helm or Operator installation methods on {ocp}, this feature is enabled by default.
-* Red Hat supports scanning for RHCOS nodes only if you are using {ocp} 4.10 and later.
+* Red Hat supports scanning for RHCOS nodes only if you are using {ocp} 4.10 and later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

Comment:
1. Not "Red Hat supports" but "RHACS supports", right?
2. Does it make sense to have the same item twice on the page?
https://60483--docspreview.netlify.app/openshift-acs/latest/operating/manage-vulnerabilities/scan-rhcos-node-host.html

![image](https://github.com/openshift/openshift-docs/assets/537715/7d475de1-18f3-4309-8200-83fff8c97b12)